servers/sunrpc/program.go

```diff
@@ -44,13 +52,20 @@ func registerProgram(server *rpc.Server, program Program, port int) error {
 	// Create procedure number to procedure name mappings for sunrpc codec
 	typeName := reflect.Indirect(reflect.ValueOf(program)).Type().Name()
 	for _, procedure := range program.Procedures() {
+		llogger := logger.WithFields(log.Fields{
+			"proc":    procedure.Name,
+			"procnum": procedure.Number,
+		})
+		llogger.Debug("registering sunrpc procedure")
+
 		err = sunrpc.RegisterProcedure(
 			sunrpc.ProcedureID{
-				program.Number(),
-				program.Version(),
-				procedure.Number,
+				ProgramNumber:   program.Number(),
+				ProgramVersion:  program.Version(),
+				ProcedureNumber: procedure.Number,
 			}, typeName+"."+procedure.Name)
 		if err != nil {
+			llogger.WithError(err).Error("failed to register sunrpc procedure")
```

Comment:
nit: The caller already logs this.

Note to self: Make [sunrpc.DumpProcedureRegistry()](https://godoc.org/github.com/prashanthpai/sunrpc#DumpProcedureRegistry) to take a logger/writer.